utils/dsu_test.go

```diff
@@ -86,10 +86,7 @@ func Test_dsuParsePreviewBytes(t *testing.T) {
 func Test_findDSUInventoryCollector(t *testing.T) {
 	invb := "invcol_5N2WM_LN64_20_09_200_921_A00.BIN"
 
-	tmpDir, err := os.MkdirTemp("/tmp", "ironlibtest")
-	if err != nil {
-		t.Error(err)
-	}
+	tmpDir := t.TempDir()
```

Comment:
TIL: t.TempDir() is a thing

Reply:
test helpers in stdlib always catch my eye